packages/web-app-files/tests/unit/components/SideBar/Details/FileDetailsMultiple.spec.ts

```diff
@@ -46,13 +46,15 @@ describe('Details Multiple Selection SideBar Item', () => {
   it('should display information for two selected folders', () => {
     const { wrapper } = createWrapper([folderA, folderB])
     expect(wrapper.find(selectors.selectedFilesText).text()).toBe('2 items selected')
+    // FIXME: Check for "preserveWhiteSpace" compiler option for tests
```

Comment:
Remove comment, it's fixed. 

We should keep the compiler settings in a file shared between vite and jest config tho.